SPECS/dracut/00-hostonly.conf

```diff
@@ -0,0 +1,4 @@
+# kdump currently uses the host system's initrd when enrolling a crash kernel
```

Comment:
Curious what would change if we had #9333 to regenerate the crash kernel initrd rather than using the host system's version

Reply:
I think we would make the kdump initrd gen logic pass -H to explictly turn on hostonly. We would then not install hostonly.conf in our image configs.

Reply:
Makes sense - I'll add this in my kdump changes